client/js/components/procedure/SegmentsList/CustomSearch.vue

```diff
@@ -41,6 +43,7 @@
             v-if="isLoading === false">
             <dp-checkbox
               v-for="({label, value}, i) in fields"
+              :data-cy="'customSearch:' + label"
```

Comment:
`label` may contain whitespace, so i think it will be safer to use `value` instead.

Reply:
https://github.com/demos-europe/demosplan-core/pull/2587/commits/26e1e9ee847358fab7a3e1b2bab8a82f0523deb5